pkg/cmd/experimental/ipfailover/ipfailover.go

```diff
@@ -155,10 +155,18 @@ func Run(f *clientcmd.Factory, options *ipfailover.IPFailoverConfigCmdOptions, c
 		return err
 	}
 
-	if len(options.ServiceAccount) == 0 {
-		return fmt.Errorf("you must specify a service account for the ipfailover pod with --service-account, it cannot be blank")
+	if options.VRRPIDOffset < 0 || options.VRRPIDOffset > 255 {
+		return fmt.Errorf("The vrrp-id-offset must be in the range 0..255")
```

Comment:
No. The real range is 1..255, The code changes 0 to 1 so the valid input range is 0..255